backend/minesearch/api/routes/consolidated_results.py

```
@@ -466,27 +588,90 @@ async def get_consolidated_results(
+                'last_updated': mine_data['last_updated'].isoformat() if mine_data['last_updated'] else None,
+                'total_sources': mine_data['total_sources']
+            }
+            
+            # PHASE 1.2: Strukturierte Datenfelder mit globalen Quellenreferenzen
+            structured_fields = {}
+            for field_name, field_breakdown in detailed_breakdown.items():
+                # Nur echte Datenfelder, keine Metadaten
+                metadaten_felder = ['Mine', 'Land', 'Zuverlässigkeit', 'Modelle', 'Letzte Aktualisierung', 'Details']
+                if field_name not in metadaten_felder:
+                    # Sammle alle globalen Quellenreferenzen für dieses Feld
+                    field_global_sources = []
```

Ruff (0.12.2)

Local variable `field_global_sources` is assigned to but never used

Remove assignment to unused variable `field_global_sources`

(F841)